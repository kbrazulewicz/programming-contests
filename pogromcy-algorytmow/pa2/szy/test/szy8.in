8
39
22092970
19050852
10097991
22960535
24336991
6431252
6489172
14030771
21306414
18611374
11173722
3227265
23619826
11347304
14042348
9403971
10121125
16263269
13128545
20010935
12434525
13799444
20080312
7773122
8883463
17003581
24360104
17674464
19929941
21815380
18345262
8455477
3285108
2197730
19085584
24221314
370149
8548051
17790104
364569739
