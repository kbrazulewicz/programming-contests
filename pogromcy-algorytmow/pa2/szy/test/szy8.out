010110100101111001010101111101001011111
